documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -795,13 +801,59 @@ EOF
 <5> Specify the name of the `NetworkMap` CR.
 <6> Specify a storage mapping even if the VMs to be migrated are not assigned with disk images. The mapping can be empty in this case.
 <7> Specify the name of the `StorageMap` CR.
-<8> By default, virtual network interface controllers (vNICs) change during the migration process. As a result, vNICs that are configured with a static IP linked to the interface name in the guest VM lose their IP.
+<8> Generates a network interface name for a target VM that overrides the template set in the plan.
```

Comment:
8 is the template in the plan, it will not overrides itself ...

Reply:
@yaacov thought it overrides the default. Will fix.